### PR TITLE
ocrvs-2562  Fixed some UI issue in application config

### PR DESCRIPTION
**Pull request by sadmananik**
Screenshot from 2022-04-01 11-47-36

### CODE REVIEW

**Comment by sadmananik** [comment]
@euanmillar The height of each row can not be changed here. Some of our code uses `ListView` component. It shows auto height.

File: packages/client/src/views/SysAdmin/Config/utils.ts [hunk 1]
```diff
@@ -113,7 +113,7 @@ export const getCurrencySelectOptions = () => {
 }
 
 export const getFormattedFee = (value: string) => {
-  value = value.replace(/\,/g, '')
+  value = value.replace(/,/g, '')
```
Comment (rikukissa) [inline packages/client/src/views/SysAdmin/Config/utils.ts:116]: Could you make this a new variable so we wouldn't override the input parameter? I know you didn't write it like this initially, but a good place to clean things up a bit :)

File: packages/client/src/views/SysAdmin/Config/utils.ts [hunk 2]
```diff
@@ -125,7 +125,14 @@ export const getFormattedFee = (value: string) => {
...
     } else {
-      return value ? Number(value).toLocaleString() : EMPTY_STRING
+      const intValue = value.split('.')
+      return !value
+        ? EMPTY_STRING
+        : intValue[1]
+        ? Number(intValue[0])
+            .toLocaleString()
+            .concat('.' + intValue[1])
+        : Number(intValue[0]).toLocaleString()
```
Comment (rikukissa) [inline packages/client/src/views/SysAdmin/Config/utils.ts:135]: I find nested ternaries a bit difficult to read. Would something like this work?

```suggestion
      if(!value) {
        return EMPTY_STRING
      }
      
      if(intValue[1]) {
        return Number(intValue[0])
            .toLocaleString()
            .concat('.' + intValue[1])
      }

      return Number(intValue[0]).toLocaleString()
```

**Review by rikukissa — APPROVED** [review]

File: packages/client/src/views/SysAdmin/Config/utils.ts (hunk 1 above)
Comment (sadmananik) [reply packages/client/src/views/SysAdmin/Config/utils.ts:116]: Yeah. Sure thing.

File: packages/client/src/views/SysAdmin/Config/utils.ts (hunk 2 above)
Comment (sadmananik) [reply packages/client/src/views/SysAdmin/Config/utils.ts:135]: Yes. Of course.

**Pull request merged**